doc/PostgreSQL_backend.txt

```diff
@@ -0,0 +1,85 @@
+PostgreSQL Backend
+==================
+
+The PostgreSQL backend is not for everyone. It requires some experience with
+PostgreSQL or a willness to learn. This is not something for the casual user.
+
+Postgresql Minetest Forum
+-------------------------
+https://forum.minetest.net/viewtopic.php?f=3&t=12851
+
+Status
+------
+This is an initial release, with improvements, and enhancements to come. Right
+now there is little more than the blocks table, but performance has been good.
+In time there will be mods that will make more use of the database, rather than
+flat files.
+
+Todo
+----
+- Adapt rollback to work with PostgreSQL, since rollback normally used SQLite3
+- Adapt minetestmapper to use PostgreSQL
+- Test under Windows
+- Adapt mods to use PostgreSQL rather than flat files, when possible
```

Comment:
1 & 4, I agree.
2 I'm actually almost done with the minetestmapper. Should be good to go in a day or two.
3 I also agree. Just fired up a Windows VM for testing. I will test in the next 24 hrs.